js/data.js

```diff
@@ -57,6 +57,6 @@ const createAd = () => {
   };
 };
 
-const similarAdd = Array.from({length: 10}, createAd);
+const createSimilarAdd = () => Array.from({length: 10}, createAd);
```

Comment:
```suggestion
const createSimilarAdd = (count) => Array.from({length: count}, createAd);
```
вот так вообще красиво будет